### ISSUE TITLE
Nav Menu doesn't work with Keyboard navigation

### ISSUE THREAD

**Issue by thekvd** [deadhanddesign/CollageMaterial#96]
Tab selects the top level button, but doesn't continue into the menu itself. This affects the Footer Nav as well.

**Comment by thekvd** [comment]
Tested after updates from Material Web to this component.  Is working as of now.
